src/qt/intro.cpp

```diff
@@ -182,9 +223,27 @@ void Intro::setDataDirectory(const QString &dataDir)
     }
 }
 
-bool Intro::showIfNeeded(bool& did_show_intro, bool& prune)
+int64_t Intro::getPruneMiB() const
 {
-    did_show_intro = false;
+    switch (ui->prune->checkState()) {
```

Comment:
It feels like oneliner `return (ui->prune->checkState() == Qt::Checked) ? PruneGBtoMiB(m_prune_target_gb) : 0` would look simpler (or just `if`/`else`) instead of "full-blown" `switch`..?

Reply:
n/a to this PR